pkgs/development/python-modules/pulsectl-asyncio/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPythonPackage, fetchPypi, lib, pulsectl }:
+
+buildPythonPackage rec {
+  pname = "pulsectl-asyncio";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-qBMIH/HOnt9SXrnrGtZTG4fOb0fftR7mBaU7K4mcYvo=";
+  };
+
```

Comment:
If you want to mark it as not for python2 you can use something like: `disabled = pythonOlder "3.7";`